addons/account/static/src/components/product_label_section_and_note_field/product_label_section_and_note_field.js

```diff
@@ -46,6 +46,11 @@ export class ProductLabelSectionAndNoteListRender extends SectionAndNoteListRend
 
         return activeColumns;
     }
+
+    isCellReadonly(column, record) {
+        let isReadonly = super.isCellReadonly(column, record);
+        return ["cancel", "done", "posted"].includes(record.evalContext.parent.state) && isReadonly;
+    }
```

Comment:
Do we need this method?

Reply:
Thanks for the feedback and sorry for not checking your comment :blush: 
Which brings me to follow up on your suggestion to either:
- Add a comment in the code
- Add an explanation in your commit message
to give some context as to why it's being done.
Could be useful for those, like me, who did not take the time to check the PR :laughing: